style_editor.html

```diff
@@ -8,14 +8,167 @@
   <title>ProffieOS Style Editor</title>
 <style>
 
-.button_on {
-  background: lightblue;
+/* Make page elements non-selectable with drag box */
+/* Took a vote, enabled won, but disabled for now. */
+/** {
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}*/
+
+/* Box sizing helps keep buttons uniform in
+size when padding and margins may change */
+* {
+  box-sizing: border-box;
+}
+
+/* Submit, copy, etc... */
+.middle-btn-container {
+  white-space: nowrap;
+}
+/* Expand and Layerize buttons */
+.middle-btn-container .button-off,
+.middle-btn-container .button-off:hover {
+  background-color: lightgray;
+  color: darkgray;
+}
+.middle-btn-container .button-on {
+  padding: 2px;
+  border: 2px solid dodgerblue;
+}
+/* Other buttons */
+button,
+input[type="button"] {
+  background-color: #ccc;
+  color: black;
+  border-radius: 2px;
+  padding: 3px;
+  font-size: 14px;
+  border: 1px solid #bbb;
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-weight: normal;
+}
+button:hover,
+select:hover,
+input[type="button"]:hover {
+  background-color: #999;
+}
+button:active,
+input[type="button"]:active {
+  background-color: dodgerblue;
+}
+/* Power and Rotate latching buttons */
+.page-left .button-latched,
+.page-left .button-latched:hover {
+  background-color: dodgerblue;
+}
+/* Lockup and More Effects Drop Downs */
+select {
+  background-color: #ccc;
   color: black;
+  border-radius: 5px;
+  padding: 2px 0px;
+  font-size: 14px;
 }
 
-.button_off {
-  background: lightgray;
+/* Prevent Safari default style takeover at default zoom level */
+@media not all and (min-resolution:.001dpcm) { @media {
+  select {
+  border: 1px solid #ccc;
+  }
+}}
+.page-left {
+  vertical-align: top;
+  width: 50%;
+}
+.page-right {
+  vertical-align: top;
+ }
+ /* Blade Preview area */
+.canvas-style {
+  width: 256px;
```

Comment:
Having the canvas be a fixed size in pixels is not good.
Might get resized below anyways, but if that's the case, then why bother?

Reply:
This, like others, were existing inlined styling that I just moved to CSS. If it can go, great. Removing it does not seem to make any difference as I just tested.
Adding commit.

Reply:
Then let's remove it.

Reply:
Done.